tools/torch_quant/torch_quant/quantizer.py

```diff
@@ -27,16 +31,43 @@ class Backend(Enum):
     FBGEMM = 2
 
 
+DEFAULT_ACT_OB_CTR: Dict[Backend, Callable[..., Observer]] = {
```

Comment:
1. `qscheme` is not set which will lead the `symmetric` of `MinMaxObserver` always `False` and further leads the wrong scales and zero_points.
2. Not a change introduced by this pull request but will affect accuracy, [here](https://github.com/alibaba/BladeDISC/blob/6c270d03d2521acb3a999f7d9dc8ad55efacfffb/tools/torch_quant/torch_quant/observer.py#L93), the zero point should to int data type.

Reply:
And better to add some uts to check the scale & zero_point after the calibration step

Reply:
> 

1. OK, qscheme is specified for each backend observer constructors in the updated commit.
2. zero point will be implicitly rounded when we copy it to a tensor with dtype=int32?

Reply:
there is a very basic test at [test_min_max_observer.py](https://github.com/alibaba/BladeDISC/blob/6c270d03d2521acb3a999f7d9dc8ad55efacfffb/tools/torch_quant/tests/test_min_max_observer.py#L19). I'll add a todo for more tests.